components/AccountWidget/AccountWidget.jsx

```diff
@@ -12,7 +12,7 @@ const AccountWidget = ({ loading, error, player }) => (
     {error && <div >Error</div>}
     {!error && !loading && player ? (
       <div>
-        <li><Link to={`/players/${player.account_id}`}>Profile</Link></li>
+        <li><Link to={`/players/${player.profile.account_id}`}>Profile</Link></li>
```

Comment:
Assuming this player is generated from metadata property `user` (which it should be!) it should not be wrapped in profile.

If you are using the state holding the current player page then this will link to the profile of whatever player you have loaded ATM.